lib/cc/analyzer/engines_config_builder.rb

```diff
@@ -11,11 +11,11 @@ class EnginesConfigBuilder
         :container_label,
       )
 
-      def initialize(registry:, config:, container_label:, source_dir:, include_paths:)
+      def initialize(registry:, config:, container_label:, source_dir:, include_paths_builder:)
```

Comment:
Gotcha, I wasn't aware that it wasn't using EnginesRunner. I'll work on moving this out.